docs/src/operations/requirements.md

```diff
@@ -19,8 +19,8 @@ transaction for each block the validator agrees with, which can cost up to
 The hardware recommendations below are provided as a guide.  Operators are encouraged to do their own performance testing.
 
 - CPU
-  - 12 cores / 24 threads, or more
-  - 2.8GHz base clock speed, or faster
+  - 16 cores / 24 threads, or more
```

Comment:
I'm kind of curious now, going to go peak around metrics and see if I can find any 12 core / 24 thread validators who are running "well"

Reply:
Switched back to 12 / 24 per above in d7a3a7aa8e